config/config.json

```diff
@@ -112,6 +112,20 @@
           "dh_price_factor" : "2",
           "trac_price_in_base_currency" : "0.4",
           "plugins": []
+        },
+        {
+          "blockchain_title": "Polygon",
+          "network_id": "polygon:testnet",
```

Comment:
We should not commit testnet implementation in development part of configuration. If we need it for development purposes we can copy it from testnet part of configuration

Reply:
We already have it for xDai. Do you suggest to remove it as well?

Reply:
Yes let's remove it both